markdown/chapters/meandeck/simple-piles.md

```diff
@@ -0,0 +1,340 @@
+---
+authors: angrybacon, Labman13, 7TeenWriters, Doishy
+order: 1
+title: Simple Piles
+---
+
+## Table of Contents
+
+## Preamble
+
+The goal of all Magic decks is to win the game. :card[Doomsday] aims to do so by
+casting its namesake card and constructing a pile of 5 cards tailored to the
+current game state. This version of the deck will almost always try to use these
+5 cards to win via :card[Thassa's Oracle].
+
+## Pile Construction Steps
+
+As described in the last chapter, a pile in this context is what we use to
+describe the stack of five cards that comprise your library after the
+resolution of Doomsday. This is terminology that will be used throughout the
+Wiki.
+
+There are three steps that need to occur to enable you to win (most of the time)
+
+1. Resolve Doomsday
+1. Draw into your pile enough so that the Oracle's trigger can win you the game
+1. Resolve Thassa's Oracle
+
+:::row{variant=CENTERED}
+DD
+Pnd
+TO
+:::
```

Comment:
Why was Ponder added?

Reply:
Follows the 3 steps listed.

DD - Resolve DD
Pnd - Draw through pile
TO - Win the game.

Reply:
Would it be clearer to the reader if we used an actual draw spell (I didn't get it at first), especially since 2 cantrip effects are used below, including Ponder already

Reply:
We've been yelled at for using IU/Pdt/NW too much already. 
Maybe Consider?